### PR TITLE
#3308 #3307 Fix My Assets 

### PR DESCRIPTION
**Pull request by drakkein**
SSH token should now be showed 
image

**Testing notes**
Both #3308 and  #3307 are fixed here

### CODE REVIEW

File: packages/web/app/test-e2e/portfolio/TokenClaim.spec.e2e.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        { submit: false },
+      );
+
+      cy.get(tid("modals.tx-sender.user-claim-flow.summary.accept")).click();
+
+      acceptWallet();
+
+      cy.get(tid("modals.tx-sender.user-claim-flow.success")).should("exist");
+      cy.get(tid("modals.tx-sender.user-claim-flow.success.go-to-portfolio")).click();
+
```
Comment (desfero) [inline packages/web/app/test-e2e/portfolio/TokenClaim.spec.e2e.ts]: Would be good to add `assertPortfolio` here IMO

File: packages/web/app/test-e2e/portfolio/TokenClaim.spec.e2e.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      clearPendingTransactions: true,
+    }).then(() => {
+      goToPortfolio();
+
+      cy.get(tid(`modals.portfolio.portfolio-asset-action.claim-${ETO_ID}`)).click();
+
+      fillForm(
+        {
+          readDocuments: {
+            type: "checkbox",
```
Comment (desfero) [inline packages/web/app/test-e2e/portfolio/TokenClaim.spec.e2e.ts]: use `toggle` type, `{false: true}` is really hard to read

File: packages/web/app/test-e2e/portfolio/TokenClaim.spec.e2e.ts [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+import { acceptWallet } from "../utils/index";
+
+describe("Token claim", () => {
+  it("claimed token should show in My Assets table", () => {
+    const ETO_ID = etoFixtureAddressByName("ETOInPayoutState");
+
+    loginFixtureAccount("INV_HAS_EUR_HAS_KYC", {
+      kyc: "business",
+      clearPendingTransactions: true,
+    }).then(() => {
```
Comment (desfero) [inline packages/web/app/test-e2e/portfolio/TokenClaim.spec.e2e.ts]: there is no need to `then` here we should await nesting when it's possible

File: packages/web/app/test-e2e/portfolio/TokenClaim.spec.e2e.ts [hunk 4]
```diff
@@ -0,0 +1,38 @@
+import { etoFixtureAddressByName, goToPortfolio, loginFixtureAccount, tid } from "../utils";
+import { fillForm } from "../utils/forms";
+import { acceptWallet } from "../utils/index";
```
Comment (desfero) [inline packages/web/app/test-e2e/portfolio/TokenClaim.spec.e2e.ts]: Remove `index`, you even can combine all imports here into one

File: packages/web/app/test-e2e/portfolio/TokenClaim.spec.e2e.ts [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+      cy.get(tid("modals.tx-sender.user-claim-flow.summary.accept")).click();
+
+      acceptWallet();
+
+      cy.get(tid("modals.tx-sender.user-claim-flow.success")).should("exist");
+      cy.get(tid("modals.tx-sender.user-claim-flow.success.go-to-portfolio")).click();
+
+      cy.get(tid(`portfolio-my-assets-token-${ETO_ID}`)).should("exist");
+      cy.get(tid(`past-investments-${ETO_ID}`)).should("exist");
```
Comment (desfero) [inline packages/web/app/test-e2e/portfolio/TokenClaim.spec.e2e.ts]: Let's also assert that it's not in the `RESERVED ASSETS` anymore

File: packages/web/app/modules/investor-portfolio/selectors.ts [hunk 6]
```diff
@@ -66,7 +66,7 @@ export const selectMyAssets = (state: IAppState): TEtoWithCompanyAndContract[] |
   const etos = selectEtos(state);
 
   if (etos) {
-    return etos.filter(eto => eto.contract && eto.contract.timedState === EETOStateOnChain.Payout);
+    return etos.filter(eto => eto.contract);
```
Comment (desfero) [inline packages/web/app/modules/investor-portfolio/selectors.ts]: Hm, this one is probably bad for performance as `selectMyAssets` is also used in `loadTokensData`. Maybe we can filter `Payout` and `Claim` state only (please confirm with @rudolfix )

Comment (rudolfix) [reply packages/web/app/modules/investor-portfolio/selectors.ts]: this was my quick fix locally to see claimed assets
`return etos.filter(eto => eto.contract && [EETOStateOnChain.Payout, EETOStateOnChain.Claim].indexOf(eto.contract.timedState));`

Comment (desfero) [reply packages/web/app/modules/investor-portfolio/selectors.ts]: This is what we should use in this case IMO, but @drakkein, please use `includes` over `indexOf`. Maybe even for readability `return etos.filter(isOnChain).filter(eto => [EETOStateOnChain.Payout, EETOStateOnChain.Claim].includes(eto.contract.timedState))`

**Pull request merged**